test/services/user/registration.test.js

```diff
@@ -208,6 +218,108 @@ describe('registration service', () => {
 			});
 	});
 
+	it('fails if user is trying to register with roles other than student/employee/expert', async () => {
+		const email = `max${Date.now()}@mustermann.de`;
+		let hash;
+		let user;
+		const hashData = {
+			toHash: email,
+			save: true,
+		};
+		return app
+			.service('hash')
+			.create(hashData)
+			.then((newHash) => {
+				hash = newHash;
+				return userModel.create({
```

Comment:
I would like if can use https://github.com/hpi-schul-cloud/schulcloud-server/blob/develop/test/services/helpers/testObjects.js  / testObjects for some parts. It will cleanup the database in the end (await testObjects.cleanup();). Otherwise you must cleanup it manual over the way that you save all created ids and remove the docs later.

Reply:
done